doc/samples/SqlConnectionStringBuilder2.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Data;
+using Microsoft.Data.SqlClient;
+
+// <Snippet1>
```

Comment:
I see these Snippet XML tags are not consistently located, and could be they're used for extracting content from this file, let's make them consistent and include all file content (mainly the using M.D.S)